internal/resources/metal/vrf_bgp_dynamic_neighbor/resource_schema.go

```diff
@@ -0,0 +1,58 @@
+package vrfbgpdynamicneighbor
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/int64planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+func resourceSchema(_ context.Context) schema.Schema {
+	return schema.Schema{
+		Description:         "This resource manages BGP dynamic neighbor ranges for an Equinix Metal VRF",
+		MarkdownDescription: "This resource manages BGP dynamic neighbor ranges for an Equinix Metal VRF, but with markdown",
```

Comment:
Need better descriptions here, and we do not need `MarkdownDescription` if we aren't going to include markdown; `Description` feeds into the docs if `MarkdownDescription` isn't specified.